modules/_labs/header/react/lib/parts/SearchBar.tsx

```diff
@@ -80,6 +80,11 @@ export interface SearchBarProps extends GrowthBehavior, React.FormHTMLAttributes
    * @default true
    */
   showClearButton?: boolean;
+  /**
+   * Height of the Search Bar in pixels
+   * @default: 40
+   */
+  height?: number;
```

Comment:
The `SearchBar` height does need to be configurable depending on different contexts (Full Headers vs. Dub Headers for example). Generally speaking, Canvas will have an opinion on form inputs' dimensions. This is to make sure they work well together. However, I don't think this should include special inputs like `SearchBar` since they exist in the context of something else (in this case, a header).

Since `SearchBar` not officially supported as a standalone component, this API change will also probably remain under the radar and should be changeable (if we decide to remove it, fix it to a set of values, etc.)